Cargo.toml

```diff
@@ -11,16 +11,26 @@ documentation = "https://docs.rs/raft"
 description = "The rust language implementation of Raft algorithm."
 categories = ["algorithms", "database-implementations"]
 edition = "2018"
+build = "build.rs"
+
+[build-dependencies]
+protoc-rust = "~2.0-2.2"
+regex = "1.1"
+protobuf-build = { git = "https://github.com/nrc/kvproto-build" }
```

Comment:
I've published version 0.2.0 which should be used here. I don't think you should need the other build deps

Reply:
And now 0.3 (has some more implementations which are needed in TiKV)